invenio_records_resources/services/records/results.py

```diff
@@ -264,6 +264,14 @@ def get_value_service(self, value):
         """Return the value and the service to fetch the referenced record."""
         return None, None
 
+    @abstractmethod
+    def tombstone(self, value):
```

Comment:
The datamodel is known by the service layer, would it make sense to have something like `service.tombstone(value)`? There would not be need for perm checks or anything, just to encapsulate correctly in the service and make it "datamodel" aware (see more in the communities PR, about hardcoding the dict).